docs/design/enforceHostNetwork.md

```diff
@@ -0,0 +1,32 @@
+# Enforcing host network design document
+
+By default, the ceph-csi controller plugin operates on the pod network
+but under some circumstances, like like setups with a dedicaated storage network, 
+where the pod network cannot connect to the ceph cluster,
+it is necessary 
+to run the Ceph-CSI controller plugin pods on the host network.
```

Comment:
I believe there is some wrapping issue here

Reply:
changed